en/06_Project_Governance/05_Major_release_policy.md

```diff
@@ -0,0 +1,135 @@
+
+### End-of-life
+
+A Silverstripe CMS major release line will be end-of-life once its follow up major release line exit active development.
+
+End-of-life major release do not receive any security fixes or bug fixes. 
+
+## PHP Support commitments
+
+Silverstripe CMS major releases tracks PHP releases. 
+
+The Silverstripe CMS release cycle is built around the assumptions that:
+- New PHP releases are shipped at the end of November on an annual basis.
+- PHP releases are in full support for two years with 1 year of limited support. 
+
+At launch, a Silverstripe CMS major release support all PHP version in full support. PHP versions in limited support are not supported at launch by new Silverstripe CMS majors.
+
+Following the initial release of a Silverstripe CMS major, we will aim to add forward compatibility for the next PHP release. e.g.: Silverstripe CMS 5 at launch will support PHP 8.1 and PHP 8.2. Will aim to add support for an eventual PHP 8.3 release to CMS 5.
+
+We try our best not to drop support for older PHP releases within Silverstripe CMS major release line even if those PHP release reach end-of-life.
+
+## Dependency management
+
+At launch new Silverstripe CMS majors aim to have all PHP and JS dependencies on their latest stable version. For dependencies offering a *Long Term Support* version, the latest LTS of that dependency will be preferred.
+
+The Silverstripe CMS development team will define a list of "fixed dependencies" for each Silverstripe CMS major release line. *Fixed dependencies* will not changed for the lifetime of a Silverstripe CMS major release line.
+
+Support for newer major releases of fixed dependencies will not be retroactively added to Silverstripe CMS major release line, even if the currently used version is end-of-life. The only exception is if the upgrade is necessary to fix a high impact bug or security vulnerability. We’ll only support 1 major release line of each fixed dependency.
+
+Non-fixed dependencies may be upgraded to new major releases or swap out altogether within a Silverstripe CMS release line.
+
+### Symfony support commitments
+
+Symfony is a collection of common PHP libraries Silverstripe CMS uses. At launch, Silverstripe CMS majors will track the latest stable major of Symfony dependencies. A Silverstripe CMS major release line will not add support for later Symfony major release.
+
+## Commercially supported module
+
+Commercially supported modules are reviewed with each new major release. At launch, the list of commercially supported modules for that release line is explicitly defined.
+
+New modules maybe added to the list of commercially for that major release line during its lifespan. We aimed not to withdraw support for modules for the duration of that major release life span.
+
+Modules that are supported for one major release line may not be supported for follow up release line.
+
+## Expected Major releases timeline
+
+(Put a pretty Graphic here)
```

Comment:
That page will probably only show current releases ... not future ones.

Reply:
Maybe it will make more sense once this graphic exists, then?
I think this PR probably should be in draft still without this graphic though. We can hardly merge it as is.

Reply:
Please either add a graphic or remove this placeholder text

Reply:
Pretty picture has been added.